docs/fidesops/docs/guides/saas_connectors.md

```diff
@@ -0,0 +1,79 @@
+# How-To: Connect to SaaS Applications
+
+## What is a SaaS connection?
+
+A SaaS (Software as a Service) connection is a connection type within Fidesops that allows a user to connect to a SaaS application (e.g., Mailchimp, Stripe, Slack, etc.) and execute data access and erasure requests against that application. These connections use functionality introduced in earlier sections ([ConnectionConfigs](database_connectors.md#creating-a-connectionconfig-object) and [Datasets](datasets.md)) but also use a new [SaaS configuration](saas_config.md) specification to define how to connect to specific SaaS applications.
+
+## Supported SaaS applications
+
+The current implementation of the SaaS framework can support any SaaS application that uses these features:
+
+- Basic and bearer authentication
+- Data access via HTTP GET requests
+- Erasure via HTTP PUT requests
+
+The following features are planned for future releases and will allow for the configuration of broader types of connections:
+
+- OAuth 2.0 authentication
+- Pagination based on headers and response contents
+- Retry logic based on status codes and response contents
+
+Full [examples](https://github.com/ethyca/fidesops/tree/main/data/saas) of a valid SaaS config and Dataset are currently available for Mailchimp.
+
+## How to configure a SaaS connector
+
+For convenience we've included a SaaS Connector [Postman](../postman/using_postman.md) collection to execute the necessary steps to configure a SaaS connector.
+
+1. Create a ConnectionConfig of type `saas`
+```
+PATCH api/v1/connection
+
+[
+  {
+    "name": "SaaS Application",
+    "key": {{saas_key}},
```

Comment:
Elsewhere we've used single `{}` to indicate variable substitution. It would be good to be consistent here. I think the main place we use `{{}}` is in the Postman collection